asm/non_matchings/code_80091750/func_8009F5E0.s

```diff
@@ -597,10 +597,10 @@ glabel L8009FD90
 glabel L8009FE9C
 /* 0A0A9C 8009FE9C 3C028019 */  lui   $v0, %hi(D_8018EDF3) # $v0, 0x8019
 /* 0A0AA0 8009FEA0 8042EDF3 */  lb    $v0, %lo(D_8018EDF3)($v0)
-/* 0A0AA4 8009FEA4 3C03800F */  lui   $v1, %hi(gGameModeRowSelectionForNumPlayers) # 0x800f
-/* 0A0AA8 8009FEA8 3C06800F */  lui   $a2, %hi(gGameModeSubMenuRowSelectionForNumPlayers) # 0x800f
+/* 0A0AA4 8009FEA4 3C03800F */  lui   $v1, %hi((D_800E86AC - 1)) # 0x800f
+/* 0A0AA8 8009FEA8 3C06800F */  lui   $a2, %hi((D_800E86B0 - 3)) # 0x800f
```

Comment:
Any reason the naming was undone while adding the pointer arithmetic?

Reply:
They need to be two separate symbols to match, but arrays will not align at the offset addresses of the original symbols. They are often indexed by `D_8018EDF3`, which is set to the Players selected enum. But that enum is 1-indexed (1 Player is 1), so you have to subtract one to get back to zero-based indexing. So, the pointer math made sense to be. 

I used the pointer numbering just to make it easier for me to keep track of what was what. I'd probably slightly change the naming but I didn't do any documenting in this file